libraries/vendor/composer/installed.json

```diff
@@ -925,5 +871,58 @@
             "portable",
             "shim"
         ]
+    },
+    {
+        "name": "joomla/filter",
+        "version": "1.2.0",
+        "version_normalized": "1.2.0.0",
+        "source": {
+            "type": "git",
+            "url": "https://github.com/joomla-framework/filter.git",
+            "reference": "45a504f0cd6fa2312a0260955e957435ca2cedc9"
+        },
+        "dist": {
+            "type": "zip",
+            "url": "https://api.github.com/repos/joomla-framework/filter/zipball/45a504f0cd6fa2312a0260955e957435ca2cedc9",
+            "reference": "45a504f0cd6fa2312a0260955e957435ca2cedc9",
+            "shasum": ""
+        },
+        "require": {
+            "joomla/string": "~1.3",
+            "php": ">=5.3.10"
+        },
+        "require-dev": {
+            "joomla/language": "~1.0",
```

Comment:
I missed this line before merging as I was a little more focused on the input filter side. So, shouldn't the `Joomla\Language\Language` package be included in the required area rather than the required-dev now that `use Joomla\Language\Language;` is part of the output filter? If that's right wouldn't the `Joomla\Language\Language` package also need to be added to the CMS?

It seems if you actually try to use the Framework library output filter you [get a fatal error for the missing class](https://travis-ci.org/photodude/joomla-cms/jobs/112350745) ... {found while attempting to extend the JOutputFilter class in the CMS from the Framework class}